File: Android/Java/Geofencing/app/src/main/java/com/example/geofencing/Interface/InterfaceConfigGeofence.java
package com.example.geofencing.Interface;

import com.google.android.gms.maps.model.LatLng;

import java.util.List;

public interface InterfaceConfigGeofence {
    void clearMaps();
    void updateCircleRadiusGraphic(float geofenceRadius);
    void updateCircleColorGraphic(String id);
    void addMarkerGeofence(LatLng latLng, float radius);
    void showMessage(String message);
    void graphRoute(List<LatLng> latLngList, String idArea);
}

File: Android/AbuMonitor/mobile/src/main/java/com/example/comunicationwearmobile/Interface/InterfaceConfigGeofence.kt
package com.example.comunicationwearmobile.Interface

import com.google.android.gms.maps.model.LatLng

interface InterfaceConfigGeofence {
    fun clearMaps()
    fun updateCircleRadiusGraphic(geofenceRadius: Float)
    fun updateCircleColorGraphic(id: String?)
    fun addMarkerGeofence(latLng: LatLng? , radius: Float)
    fun showMessage(message: String?)
    fun graphRoute(latLngList: List<LatLng?>? , idArea: String?)
}
